visualization/app/codeCharta/ui/ribbonBar/showScenariosButton/addCustomScenario/addCustomScenario.component.spec.ts

```diff
@@ -19,25 +19,26 @@ describe("AddCustomScenarioComponent", () => {
 			providers: [
 				{ provide: ThreeCameraService, useValue: { camera: { position: new Vector3(0, 300, 1000) } } },
 				{ provide: ThreeOrbitControlsService, useValue: { controls: { target: new Vector3(177, 0, 299) } } },
-				{ provide: MatLegacyDialogRef, useValue: { close: jest.fn() } }
+				{ provide: MatDialogRef, useValue: { close: jest.fn() } }
 			]
 		})
 	})
 
-	it("should enable the user to add a custom scenario, with all properties set initially", async () => {
```

Comment:
We should fix this test or create a follow up ticket for it. I strongly prefer to immediately fix it :)

Reply:
done